### PR TITLE
Shutup ci

### PR DESCRIPTION
**Pull request by stonedDiscord**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -180,8 +180,19 @@ jobs:
         working-directory: ${{github.workspace}}/bin/
         shell: bash
         run: |
-          windeployqt .
+          windeployqt --no-quick-import --no-translations --no-virtualkeyboard .
```
Comment (oldmud0) [inline .github/workflows/build.yml:183]: `--compiler-runtime`, `--no-webkit2`, `--no-opengl-sw`
Maybe adding `--pdb` and put the pdbs in a separate zip would be helpful for testing

**Review by oldmud0 — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (stonedDiscord) [reply .github/workflows/build.yml]: pdb doesn't seem very useful as i myseld wouldn't remember which exact artifact i downloaded and neither would the testers

**Comment by stonedDiscord** [comment]
why is android suddenly failing

**Comment by oldmud0** [comment]
your cross compiler `arm-linux-androideabi-ranlib` disappeared

**Comment by stonedDiscord** [comment]
YEAH LETS GOOOOOOOO

**Review by Salanto — APPROVED** [review]
Tested the Android, Windows and Linux one. Linux one needed some manual tweaks due to mismatch Qt versions on the host, but those are confirmed to open at least.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -180,8 +196,19 @@ jobs:
...
+          windeployqt --no-quick-import --no-translations --no-virtualkeyboard --no-compiler-runtime --no-webkit2 --no-opengl-sw .
           cp ../QtApng/plugins/imageformats/qapng.dll ./imageformats/
+          rm ./imageformats/qtiff.dll
+          rm ./imageformats/qjpeg.dll
+          rm ./imageformats/qicns.dll
+          rm ./imageformats/qico.dll
+          rm ./imageformats/qsvg.dll
+          rm ./imageformats/qtga.dll
+          rm ./imageformats/qwbmp.dll
+          rm ./Qt5Svg.dll
```
Comment (oldmud0) [inline .github/workflows/build.yml:208]: I think we should keep svg for the theme diehards who use qss to set their images but this is fine for now.

**Review by oldmud0 — APPROVED** [review]
Ebic

Later we can make the clang-tidy shut up because that thing is very annoying... it should use github's review annotations API instead of review comments

**Pull request merged**